include/rnp.h

```diff
@@ -33,6 +33,7 @@
 
 #include <stddef.h>
 #include "packet.h"
+#include "../common/constants.h"
```

Comment:
I used this include as it was done previously. But I agree - we should consolidate usage of headers, but in some other PR - this one is too huge already.